lib/ipfs.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Manages the IPFS connection.
+ */
+
+const IPFS = require('ipfs')
+const ipfs = {}
+
+ipfs.connect = async function() {
```

Comment:
Please use named functions on high level APIs. Also async is unnecessary here

```
function connect () {
...
module.exports = {
 connect
}
```

Reply:
👍